Ow... I didn't expect failed

> [info] Run completed in 1 minute, 24 seconds.
[info] Total number of tests run: 632
[info] Suites: completed 180, aborted 0
[info] Tests: succeeded 632, failed 0, canceled 0, ignored 6, pending 1
[info] All tests passed.
[warn] In the last 803 seconds, 5.667 (0.7%) were spent in GC. [Heap: 0.90GB free of 1.72GB, max 3.00GB] Consider increasing the JVM heap using `-Xmx` or try a different collector, e.g. `-XX:+UseG1GC`, for better performance.
[error] (akka-stream-tests / Test / compileIncremental) Compilation failed
[error] Total time: 805 s (13:25), completed Dec 15, 2021 7:44:00 PM
Error: Process completed with exit code 1.

Build pipeline error?

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
@@ -56,6 +56,7 @@ class ActorRefSourceSpec extends StreamSpec {
     }
 
     "drop new when full and with dropNew strategy" in {
+      @nowarn("msg=deprecated")
       val (ref, sub) = Source
         .actorRef(PartialFunction.empty, PartialFunction.empty, 100, OverflowStrategy.dropNew)
         .toMat(TestSink.probe[Int])(Keep.both)

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
@@ -5,9 +5,7 @@
 package akka.stream.scaladsl
 
 import scala.concurrent.duration._
-
 import org.reactivestreams.Publisher
-
 import akka.Done
 import akka.actor.{ ActorRef, Status }
 import akka.stream.{ OverflowStrategy, _ }
@@ -15,6 +13,8 @@ import akka.stream.testkit._
 import akka.stream.testkit.Utils._
 import akka.stream.testkit.scaladsl._
 import akka.stream.testkit.scaladsl.StreamTestKit._
+
+import scala.annotation.nowarn
 
 class ActorRefSourceSpec extends StreamSpec {
 
@@ -38,6 +38,7 @@ class ActorRefSourceSpec extends StreamSpec {
 
     "buffer when needed" in {
       val s = TestSubscriber.manualProbe[Int]()
+      @nowarn("msg=deprecated")
       val ref = Source
         .actorRef(PartialFunction.empty, PartialFunction.empty, 100, OverflowStrategy.dropHead)
         .to(Sink.fromSubscriber(s))

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefSourceSpec.scala
@@ -7,8 +7,8 @@ package akka.stream.scaladsl
 import scala.concurrent.duration._
 import org.reactivestreams.Publisher
 import akka.Done
-import akka.actor.{ ActorRef, Status }
-import akka.stream.{ OverflowStrategy, _ }
+import akka.actor.{ActorRef, Status}
+import akka.stream.{OverflowStrategy, _}
 import akka.stream.testkit._
 import akka.stream.testkit.Utils._
 import akka.stream.testkit.scaladsl._
@@ -16,6 +16,7 @@ import akka.stream.testkit.scaladsl.StreamTestKit._
 
 import scala.annotation.nowarn
 
+@nowarn("msg=deprecated")
 class ActorRefSourceSpec extends StreamSpec {
 
   "A ActorRefSource" must {
@@ -38,7 +39,6 @@ class ActorRefSourceSpec extends StreamSpec {
 
     "buffer when needed" in {
       val s = TestSubscriber.manualProbe[Int]()
-      @nowarn("msg=deprecated")
       val ref = Source
         .actorRef(PartialFunction.empty, PartialFunction.empty, 100, OverflowStrategy.dropHead)
         .to(Sink.fromSubscriber(s))
@@ -56,7 +56,6 @@ class ActorRefSourceSpec extends StreamSpec {
     }
 
     "drop new when full and with dropNew strategy" in {
-      @nowarn("msg=deprecated")
       val (ref, sub) = Source
         .actorRef(PartialFunction.empty, PartialFunction.empty, 100, OverflowStrategy.dropNew)
         .toMat(TestSink.probe[Int])(Keep.both)

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowBufferSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowBufferSpec.scala
@@ -7,12 +7,13 @@ package akka.stream.scaladsl
 import scala.concurrent.Await
 import scala.concurrent.Future
 import scala.concurrent.duration._
-
 import akka.stream.BufferOverflowException
 import akka.stream.OverflowStrategy
 import akka.stream.testkit._
 import akka.stream.testkit.scaladsl._
 import akka.stream.testkit.scaladsl.StreamTestKit._
+
+import scala.annotation.nowarn
 
 class FlowBufferSpec extends StreamSpec("""
     akka.stream.materializer.initial-input-buffer-size = 1
@@ -171,6 +172,7 @@ class FlowBufferSpec extends StreamSpec("""
     }
 
     "drop new elements if buffer is full and configured so" in {
+      @nowarn("msg=deprecated")
       val (publisher, subscriber) = TestSource
         .probe[Int]
         .buffer(100, overflowStrategy = OverflowStrategy.dropNew)

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowBufferSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowBufferSpec.scala
@@ -15,6 +15,7 @@ import akka.stream.testkit.scaladsl.StreamTestKit._
 
 import scala.annotation.nowarn
 
+@nowarn("msg=deprecated")
 class FlowBufferSpec extends StreamSpec("""
     akka.stream.materializer.initial-input-buffer-size = 1
     akka.stream.materializer.max-input-buffer-size = 1
@@ -172,7 +173,6 @@ class FlowBufferSpec extends StreamSpec("""
     }
 
     "drop new elements if buffer is full and configured so" in {
-      @nowarn("msg=deprecated")
       val (publisher, subscriber) = TestSource
         .probe[Int]
         .buffer(100, overflowStrategy = OverflowStrategy.dropNew)

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/QueueSourceSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/QueueSourceSpec.scala
@@ -6,9 +6,7 @@ package akka.stream.scaladsl
 
 import scala.concurrent._
 import scala.concurrent.duration._
-
 import org.scalatest.time.Span
-
 import akka.Done
 import akka.actor.Status
 import akka.pattern.pipe
@@ -21,6 +19,8 @@ import akka.stream.testkit.TestSubscriber
 import akka.stream.testkit.scaladsl.StreamTestKit._
 import akka.stream.testkit.scaladsl.TestSink
 import akka.testkit.TestProbe
+
+import scala.annotation.nowarn
 
 class QueueSourceSpec extends StreamSpec {
   implicit val ec: ExecutionContextExecutor = system.dispatcher
@@ -227,6 +227,7 @@ class QueueSourceSpec extends StreamSpec {
 
     "return false when element was not added to buffer" in assertAllStagesStopped {
       val s = TestSubscriber.manualProbe[Int]()
+      @nowarn("msg=deprecated")
       val queue = Source.queue(1, OverflowStrategy.dropNew).to(Sink.fromSubscriber(s)).run()
       val sub = s.expectSubscription()
 
@@ -260,6 +261,7 @@ class QueueSourceSpec extends StreamSpec {
 
     "fail offer future when stream is completed" in assertAllStagesStopped {
       val s = TestSubscriber.manualProbe[Int]()
+      @nowarn("msg=deprecated")
       val queue = Source.queue(1, OverflowStrategy.dropNew).to(Sink.fromSubscriber(s)).run()
       val sub = s.expectSubscription()
       queue.watchCompletion().pipeTo(testActor)

--- a/akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala
+++ b/akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala
@@ -114,8 +114,11 @@ object OverflowStrategy {
 
   /**
    * If the buffer is full when a new element arrives, drops the new element.
+   *
+   * @deprecated Use {@link akka.stream.javadsl.Source#queue(int,akka.stream.OverflowStrategy)} instead
    */
   @deprecated("Use Source.queue instead", "2.6.11")
+  @Deprecated
   def dropNew: OverflowStrategy = DropNew(Logging.DebugLevel)
 
   /**

